docs/releases/v2015.2.rst

```diff
@@ -47,14 +62,12 @@ Site changes
         },
       }
 
-    The ``htmode`` option has been dropped, the channel width is now always set to 20MHz
-    (see https://github.com/freifunk-gluon/gluon/issues/487 for a discussion of this change).
-
```

Comment:
Please don't change that; the htmode option is gone and won't return. It is possible that we'll add support for higher channel widths again in the future, but it is not planned to make it configurable.